extensions/notebook/coverConfig.json

```diff
@@ -6,6 +6,7 @@
 		"**/integrationTest/**",
 		"**/node_modules/**",
 		"**/test/**",
+		"**/integrationTest/**",
```

Comment:
3 lines above - I added that a while back. Were you still seeing those files being instrumented?

Reply:
sigh. i'm 100% sure that i saw the integrationTests folder in the coverage reports last night. Now I can't repro. Oh well. Thanks.